package.json

```diff
@@ -1,24 +1,23 @@
 {
-  "name": "d2l-intl",
-  "version": "2.1.0",
-  "description": "D2L internationalization APIs for number, date and time formatting and parsing.",
-  "main": "dist/index.js",
-  "types": "src/index.d.ts",
-  "module": "src/index.js",
+  "name": "@brightspace-ui/intl",
```

Comment:
Scoped packages are private by default so this will be needed in order to publish:
```
  "publishConfig": {
    "access": "public"
  },
```
Side note: I believe that if the org does not have the private packages feature, the publish will fail unless public access is explicitly specified.

Reply:
Right -- thanks, fixed!